crates/turborepo-lib/src/task_graph/visitor/mod.rs

```diff
@@ -97,6 +99,11 @@ pub enum Error {
     InternalErrors(String),
     #[error("unable to find package manager binary: {0}")]
     Which(#[from] which::Error),
+    #[error(
+        "'{package}' is configured with a microfrontend.jsonc, but doesn't have \
```

Comment:
We can address this later, but this config name isn't guaranteed and is just a convention right now. But we can fix that once we have the path available for you to source.